requirements.txt

```diff
@@ -0,0 +1,5 @@
+drmaa==0.7.9
+hydra-genetics==v0.8.0
```

Comment:
There is a 0.9.0 now. Should it be v before?

Reply:
you are completely right

Reply:
v is not required for some reason 🤔